### ISSUE TITLE
Quais materiais já estão completos?

### ISSUE THREAD

**Issue by arthurcaccavo** [RoenMidnight/grimorio-trpg#4]
Cara o projeto ta MUITO bom, parabéns. Mas ficou faltando uma parte, podia ter um local que diz de que manual vem a magia, pra ser possível ter noção de quais manuais já estão completos, quais magias faltam e coisas do tipo. Se possível, seria incrível. Estou disposto a ajudar!

**Comment by oriportinho** [comment]
sim, no nosso último commit corrigimos e colocamos todas as magias do jogo, inclusive com um campo com o manual que ela se encontra e a página, porém ele ta dando um erro quando estamos tentando rodar o Jekyll. Temos em aberto melhorar o layout do site também.
